src/sentry/api/bases/organization.py

```diff
@@ -92,6 +92,25 @@ class OrganizationIntegrationsPermission(OrganizationPermission):
     }
 
 
+class OrganizationApiKeysPermission(OrganizationPermission):
+    scope_map = {
+        'GET': ['org:admin'],
+        'POST': ['org:admin'],
+        'PUT': ['org:admin'],
+        'DELETE': ['org:admin'],
+    }
+
+
+class OrganizationAuthProvidersPermission(OrganizationPermission):
+    scope_map = {
+        # Required for members view
+        'GET': ['member:read', 'org:read'],
```

Comment:
Not sure if this is the right permission, we need access to `GET auth-provider` for the Members view